### ISSUE TITLE
Network speed unit wrong on Linux

### ISSUE THREAD

**Issue by detyo** [oshi/oshi#440]
Hi,
  I have been looking into network statistics reported by OSHI on different OS-es and I noticed that on Linux, the network speed is assumed to be in bits per second, when read from `/sys/class/net/<iface>/speed`, but according to Linux [kernel documentation](https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-net) the value is in Mbits/sec. 
I tested this on Red Hat Enterprise Linux Server 6.0 (build 2.6.32-71.el6.x86_64) VM, but it seems that on a physical machine the units are the same.
For example, on the mentioned VM the file contains 1000 and OSHI displays this as:

```
Network interfaces:
 Name: eth1 (eth1)
   MAC Address: 00:50:56:9d:79:cd 
   MTU: 1500, Speed: 1 Kbps 
```

while `ethtool` returns: Speed: 1000Mb/s

Is this a bug or perhaps the interface speed unit might differ based on kernel version?

Detelin

**Comment by dbwiddis** [comment]
Probably a bug.  Should be an easy fix!

**Comment by YoshiEnVerde** [comment]
Added the fix for this in my latest PR #443

**Comment by detyo** [comment]
Thanks for fixing it! I was going to do this myself but got distracted with some high-prio tasks.

**Comment by dbwiddis** [comment]
Fixed in #443
